### ISSUE TITLE
[Discuss] Reset class style for element

### ISSUE THREAD

**Issue by terrykingcha** [alibaba/weex#397]
There are problems about reset class styles. This is the exmaple(or like this issue #218 ):

```
<template>
  <div>
    <text class="{{className}}" onclick="toggle">Hello Weex</text>
  </div>
</template>

<style>
.normal {
  font-size: 40px;
}

.hightlight {
  font-size: 40px;
  color: red;
}
</style>

<script>
module.exports = {
  data: {
    className: 'highlight'
  },
  methods: {
    toggle: function() {
      if (this.className === 'normal') {
         this.className = 'hightlight'
      } else {
        this.className = 'normal'
      }
    }
  }
}
</script>
```

When click the text, the color of the text is still `red`. Because we have no way to clear the last class style. 

So maybe we can give the style a empty value(string) to clear the style through calling 'updateStyles'.

What's your opinion?

**Comment by Jinjiang** [comment]
@Hanks10100 Please have a look at this issue.
Thanks.

**Comment by Hanks10100** [comment]
After fixed this bug in #953 ,  *jsfm* will set empty string on the style whitch need to be reset. I'm sure it works well on HTML5. I hope this feature could be fully support by native.

**Comment by Jinjiang** [comment]
Though the code merged into `jsfm` branch, maybe more discussion needed with the native part about the detail of sending default value. @Hanks10100 @cxfeng1 @zshshr 
Thanks.
